src/main/java/seedu/address/logic/parser/TagsCommandParser.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_FIND;
+
+import java.util.Optional;
+
+import seedu.address.logic.commands.TagsCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class TagsCommandParser implements Parser<TagsCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the TagsCommand
+     * and returns an TagsCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public TagsCommand parse(String args) throws ParseException {
+        if (args.trim().equals("")) {
+            return new TagsCommand(tag -> true);
+        }
+
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_FIND);
+
+        Optional<String> keyword = argMultimap.getValue(PREFIX_FIND);
```

Comment:
just to standardise with other parser classes if you want haha 

```suggestion
        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_FIND);
        
        Optional<String> keyword = argMultimap.getValue(PREFIX_FIND);
        
        if (keyword.isEmpty() && argMultimap.getPreamble().isEmpty()) {
            return new TagsCommand(tag -> true);
        }
```